tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -152,6 +152,12 @@ message TablePartitionProto {
   optional string path = 4;
 }
 
+message TablePartitionKeyProto {
```

Comment:
Yes. I mean, what is the difference between PartitionKeyProto and TablePartitionKeyProto? Their name is similar, but contents are different. I wonder their proper usage.